aws-ts-vpc-with-ecs-fargate-py/ecs-fargate-python/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pulumi>=2.0.0,<3.0.0
+pulumi-aws>=2.0.0,<3.0.0
```

Comment:
We should use >3, <4 now since that's our latest versions

Reply:
Updated